src/serlio/modifiers/PRTModifierAction.cpp

```diff
@@ -630,20 +629,13 @@ MStatus PRTModifierAction::updateDynamicEnums() {
 				for (short enumIndex = 0; enumIndex < arr_length; enumIndex++) {
 					if (stringArray[enumIndex] == nullptr)
 						continue;
-					const std::wstring_view currStringView = stringArray[enumIndex];
+					std::wstring_view currStringView = stringArray[enumIndex];
 
 					// remove newlines from strings, because they break the maya UI
 					const size_t cutoffIndex = currStringView.find_last_of(L"\r\n");
+					currStringView = currStringView.substr(0, cutoffIndex);

```

Comment:
could avoid reassigning the string_view:
```suggestion
					currStringView.remove_suffix(currStringView.length() - cutoffIndex);

```

https://en.cppreference.com/w/cpp/string/basic_string_view/remove_suffix